draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -556,7 +556,9 @@ to the one in the `vct#integrity` claim in the SD-JWT VC payload,
 
 The Type Metadata document MUST be a JSON object. The following properties are
 defined:
-
+* `vct`
+  * REQUIRED. The verifiable credential type described by this type metadata document.
+    The value MUST be identical to the `vct` claim used in the SD-JWT VC payload.
```

Comment:
Is this "The value MUST be identical to the `vct` claim used in the SD-JWT VC payload" problematic with metadata inheritance?